psalm.xml

```diff
@@ -32,6 +37,10 @@
         </InternalMethod>
     </issueHandlers>
 
+    <stubs>
+        <file name=".psalm-stubs.meta.php"/>
+    </stubs>
+
```

Comment:
I've found that using a `.php` extension on psalm stubs can interfere with type inference in the editor. Psalm doesn't care what the filename extension is, so I use an extension like `.php.stub`